symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/model/ActivityType.java

```diff
@@ -0,0 +1,18 @@
+package com.symphony.bdk.core.activity.model;
+
+import org.apiguardian.api.API;
+
+/**
+ * Type of an activity.
+ */
+@API(status = API.Status.EXPERIMENTAL)
+public enum ActivityType {
+  /**
+   * Message sent in the chat.
+   */
+  command,
```

Comment:
do we have a reason to not follow the Java style here (uppercase) (like is it used in a REST API somewhere)

Reply:
Erfff not really to be honest :) I'll rename it in my current ticket 👍